src/Kaleidoscope-LEDControl.cpp

```diff
@@ -117,8 +130,12 @@ Key LEDControl::eventHandler(Key mappedKey, byte row, byte col, uint8_t keyState
   if (mappedKey.flags != (SYNTHETIC | IS_INTERNAL | LED_TOGGLE))
     return mappedKey;
 
-  if (keyToggledOn(keyState))
-    next_mode();
+  if (keyToggledOn(keyState)) {
+    if (mappedKey.keyCode == 0)
+      next_mode();
+    else
```

Comment:
It might be worth changing the code to something like this:

```c++
if (mappedKey == Key_LEDEffectNext) {
  next_mode();
} else if (mappedKey == Key_LEDEffectPrevious) {
  prev_mode();
}
```

This guards against code that creates a key that has all the right bits, but has a keyCode other than 0 or 1, and conveys the intent better, too.